source/index.html.md

```diff
@@ -793,3 +793,369 @@ Account | Income Statement:Revenue:Product Revenue
 Department | Consolidated Only
 Time | 2019:Q1:Jan-19
 Scenario | Actuals
+
+
+# Report Templates
+
+User configuration of sheets ranges, with their corresponding rows, columns, and filters.
+
+For development purposes, a Report is a given range on a user's sheet, with row/column/filter data. A Report Template is a collection of Reports on a given sheet (aka multiple ranges on a single sheet).
+
+
+## Retrieve Report Templates
+
+```python
+import requests
+
+r = requests.get('<url>', headers={
+    'Authorization': 'example-access-token'
+})
+r.json()
+```
+
+> The above request returns a JSON structured like this:
+
+```json
+[
+    {
+        "modified_at": "2019-09-24T17:28:56.739624+00:00",
+        "created_by": { "id": "1" },
+        "id": "5",
+        "created_at": "2019-09-24T17:28:56.739514+00:00",
+        "name": "Template #2",
+        "reports": [
+            {
+                "id": "16",
+                "sheet_range": "B1:C5",
+                "rows": {
+                    //Top-level dimension id: [Row dimenion IDs]
```

Comment:
JSON can't have comments unfortunately :-(

Maybe we can add a "### Response Explanation" section to the left hand side that we can use to explain various things about the JSON response? Could maybe look like the format of the table below?

<img width="547" alt="Screenshot 2019-09-25 11 18 20" src="https://user-images.githubusercontent.com/477603/65614765-2f149300-df86-11e9-8243-205bbbc92355.png">